docs/src/main/sphinx/installation/query-resiliency.rst

```diff
@@ -0,0 +1,120 @@
+---------------------------------
+
+A fault-tolerant cluster is best suited for large batch queries. Fault-tolerant
+execution of a high volume of short-running may experience increased latency. As
+such, it is recommended to run a dedicated fault-tolerant cluster for handling
+batch operations, separate from a cluster that is designated for a higher query
+volume.
+
+Catalogs using the following connectors support fault-tolerant execution of read
+and write operations:
+
+* :doc:`/connector/delta-lake`
+* :doc:`/connector/hive`
+* :doc:`/connector/iceberg`
+* :doc:`/connector/mysql`
+* :doc:`/connector/postgresql`
+* :doc:`/connector/sqlserver`
+
+Catalogs using other connectors only support fault-tolerant execution of read
+operations. When fault-tolerant execution is enabled on a cluster, write
+operations fail on any catalogs that do not support fault-tolerant
+execution of those operations.
+
+The exchange manager may send a large amount of data to the exchange storage,
+resulting in high I/O load on that storage. You can configure multiple storage
+locations for use by the exchange manager to help balance the I/O load between
+them.
+
+Configuration
+-------------
+
+The following steps describe how to configure a Trino cluster for
+fault-tolerant execution with an S3-based exchange:
+
+1. Set up an S3 bucket to use as the exchange storage. For this example we are
+   using an AWS S3 bucket, but other storage options are described in the
+   :doc:`reference documentation </admin/fault-tolerant-execution>`
+   as well. You can use multiple S3 buckets for exchange storage.
+
+   For each bucket in AWS, collect the following information:
+
+   * S3 URI location for the bucket, such as ``s3://exchange-spooling-bucket``
+
+   * Region that the bucket is located in, such as ``us-west-1``
+
+   * AWS access and secret keys for the bucket
+
+2. For a :doc:`Kubernetes deployment of Trino </installation/kubernetes>`, add
+   the following exchange manager configuration in the
+   ``server.exchangeManager`` and ``additionalExchangeManagerProperties``
+   sections of the Helm chart, using the gathered S3 bucket information:
+
+   .. code-block:: yaml
+
+       server:
+         exchangeManager:
+           name=filesystem
+           base-directories=s3://exchange-spooling-bucket-1,s3://exchange-spooling-bucket-2
+
+       additionalExchangeManagerProperties:
+         s3.region=us-west-1
+         s3.aws-access-key=example-access-key
+         s3.aws-secret-key=example-secret-key
+
+   In non-Kubernetes installations, the same properties must be defined in an
+   ``exchange-manager.properties`` configuration file on the coordinator and
+   all worker nodes.
+
+3. Add the following configuration for fault-tolerant execution in the
+   ``additionalConfigProperties:`` section of the Helm chart:
+
+   .. code-block:: yaml
+
+       additionalConfigProperties:
+         retry-policy=TASK
+         exchange.compression-enabled=true
```

Comment:
@jhlodin https://github.com/trinodb/trino/pull/14957

Let's have this landed and make proper adjustments to this guide before landing. 

Big thanks for helping us with this.

Reply:
Sounds good to me. We'll also need to modify the FTE reference doc (https://trino.io/docs/current/admin/fault-tolerant-execution.html) to describe what settings change when enabling task retries, either in your PR or I can update it afterwards once yours is merged and we know exactly what the setting changes will be.

Reply:
Added `task.concurrency=8` to the YAML anyway, even if it's set as the default in the future it's helpful to show what properties are relevant.

Reply:
> Added task.concurrency=8 to the YAML anyway, even if it's set as the default in the future it's helpful to show what properties are relevant.

I would prefer to avoid it. It may tend to evolve over time and ideally we would like to avoid users setting anything explicitly if not necessary, as then it might not be possible to easily change the recommended settings.

Reply:
Got it, I'll remove it then. This page is intended as a "quick setup" guide anyway, so if we want to discuss more nuances for this property then the reference docs are more appropriate for that.

Reply:
Let's remove `exchange.compression-enabled=true` and `query.low-memory-killer.delay=0s` from here as well, as now those are set automatically

Reply:
@arhimondr I thought they aren't changed if the user has them explicitly configured? My thought is that it can't hurt to leave them in, in case the reader has the property configured and would then need to change its value in order to meet best practices.

Reply:
Hmm, maybe it's worth mentioning it explicitly, something like
"If you have exchange.compression-enabled, query.low-memory-killer.delay configured explicitly - remove them", or something like that?

Basically what I'm afraid of is that somebody may configure `query.low-memory-killer.delay=0s` explicitly following our recommendation, but the recommended value might need to be changed in the future.

Reply:
That makes sense to me. I removed those properties from this section, then to address the new auto-default change behavior I rewrote the paragraph on line 125 of the /admin/ guide in this PR.